docs/themes/thxvscode/assets/js/package.json

```
@@ -8,14 +8,12 @@
     "@babel/runtime": "^7.17.2",
     "babel-loader": "^8.2.3",
     "core-js": "^3.21.1",
-    "terser-webpack-plugin": "^5.3.1",
+    "terser-webpack-plugin": "^4",
```

Nit (for consistency): 
```suggestion
    "terser-webpack-plugin": "^4.0.0",
```